test/unit/mock/config_dirs/test-fixture-2-float/gems/2.3.0/gems/ordinal_array-0.2.0/ordinal_array.gemspec

```diff
@@ -0,0 +1,19 @@
+# -*- encoding: utf-8 -*-
+# stub: ordinal_array 0.2.0 ruby lib
+
+Gem::Specification.new do |s|
+  s.name = "ordinal_array"
+  s.version = "0.2.0"
+
+  s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
+  s.require_paths = ["lib"]
+  s.authors = ["Kevin Disneur"]
```

Comment:
I didn't see a license anywhere in this gem, is there any worry about redistributing this in our repository?